src/web_app/static/js/reminder.js

```diff
@@ -0,0 +1,198 @@
+
+// classes disappear after applying filters and then sorting
+// so reset them after table body rendering
+$(document).ready(function (){
+	$('#reminders-table').on('post-body.bs.table', setClasses)
+})
+
+
+// passes all filter fields to the back
+function getFilters(){
+	let course = $('#course').val()
+	let author = $('#author').val()
+	let from_date = $('#from').val()
+	let to_date = $('#to').val()
+	let posted = $('#check').is(':checked')
+	$.post('/ui/reminder/filter?name=' + course + '&author=' + author + '&from=' + from_date + '&to=' + to_date + '&posted=' + posted,
+		updateTable, 'json')
+}
+
+/*
+updates data in the table
+params: data : json { result : number of reminders
+					  rows : reminder data }
+ */
+function updateTable(data){
+	let optionsHTML = "&hellip;<div class=\"dropdown-menu menu\">\n" +
```

Comment:
koje su alternative ovom postavljanju `innerHtml` atributa u JS-u?

Reply:
Nikakva. Stvarno sam tražio načine da ne petljam po HTMLu unutar JS-a, ali nisam uspio naći kombinaciju gdje ne diram HTML i `bootstraptable` radi load. Jedina što mi je palo na pamet je da edit i delete ne budu vezani za red nego u posebnoj tablici, samo što bi ih se trebalo vezat nekako, ovo mi se činilo lakše.